lib/devices/index.js

```diff
@@ -0,0 +1,10 @@
+/*!
+ * index.js - type definitions
+ * Copyright (c) 2019, Nodari Chkuaselidze (MIT License)
+ */
+
+'use strict';
+
+/**
+ * @module device
+ */
```

Comment:
Were you planning on adding the typedefs here as part of this PR?

Reply:
I moved them in different branch that I have not published, I will update